### PR TITLE
fix(CountryFlag): set "undefined" as default code

### PR DESCRIPTION
**Pull request by silvenon**
The default code for CountryFlag is now `"undefined"` rather than `"anywhere"`, a gray flag with a question mark. This code is also used if the given code isn't supported.

 Storybook: https://orbit-silvenon-fix-default-country-flag-code.surge.sh

### CODE REVIEW

File: packages/orbit-components/src/CountryFlag/index.js [hunk 1]
```diff
@@ -76,12 +76,12 @@ StyledShadow.defaultProps = {
 };
 
 function getCountryProps(code: ?string, name: ?string): {| code: string, name: ?string |} {
```
Comment (mainframev) [inline packages/orbit-components/src/CountryFlag/index.js:78]: the line was not changed, but  I guess it should be: `getCountryProps(code?: string, name?: string): {| code: string, name: string |}`, right?

**Review by mainframev — APPROVED** [review]

File: packages/orbit-components/src/CountryFlag/index.js [hunk 2]
```diff
@@ -76,12 +76,12 @@ StyledShadow.defaultProps = {
...
-  const codeNormalized = code ? code.toUpperCase().replace("-", "_") : "ANYWHERE";
+  const codeNormalized = code ? code.toUpperCase().replace("-", "_") : "UNDEFINED";
   const countryCodeExists = codeNormalized in CODES;
 
   warning(countryCodeExists, "Country code not supported: %s", code);
-  const countryCode = countryCodeExists ? CODES[codeNormalized] : CODES.ANYWHERE;
-  const countryName = countryCode === CODES.ANYWHERE && !name ? "Anywhere" : name;
+  const countryCode = countryCodeExists ? CODES[codeNormalized] : CODES.UNDEFINED;
+  const countryName = countryCode === CODES.UNDEFINED && !name ? "Undefined" : name;
```
[the remark this answers is not in the document]
Comment (mainframev) [reply packages/orbit-components/src/CountryFlag/index.js:84]: should be ok 🤷🏻

File: packages/orbit-components/src/CountryFlag/index.js (hunk 1 above)
Comment (silvenon) [reply packages/orbit-components/src/CountryFlag/index.js]: Returned `name` can still be `null`, but I changed the rest.

**Comment by silvenon** [comment]
Disallowing `null` is another breaking change, so make sure to check it.

**Pull request merged**